reference/pointset.md

```diff
@@ -0,0 +1,22 @@
+## PointSet
+
+```
+PointSet {
+  SFNode  color   NULL  # {Color, PROTO}
+  SFNode  coord   NULL  # {Coordinate, PROTO}
+}
+```
+
+### Description
+
+The [PointSet](#pointset) node represents a 3D geometry formed by a set of 3D points specified in the `coord` field.
```

Comment:
I would remove the "represents a 3d geometry formed by" - it's not really correct and "a geometry" isn't right either.